### PR TITLE
Added city_names_transformation

### PR DESCRIPTION
**Pull request by afnanmmir**
City names transformation replaces populous city names in documents with more scarcely populated cities in order to test whether NLP models are invariant to changes in name. This means that the context should be used in order to make decisions on entities rather than the entity itself.

### CODE REVIEW

File: transformations/city_names_transformation/transformation.py [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+            data path of database of English and Spanish cities. The default is None.
+
+        Returns
+        -------
+        None.
+
+        """
+        super().__init__(seed, max_outputs=max_outputs)
+        if lang == "en":
+            self.model = spacy.load("en_core_web_sm")
```
Comment (kaustubhdhole) [inline transformations/city_names_transformation/transformation.py:193]: You might want to use spacy like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/d39f0e5f57a79937296351996617ac03ee80fb74/transformations/close_homophones_swap/transformation.py#L55).

File: transformations/city_names_transformation/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+es_core_news_sm @ https://github.com/explosion/spacy-models/releases/download/es_core_news_sm-3.0.0/es_core_news_sm-3.0.0.tar.gz
```
Comment (kaustubhdhole) [inline transformations/city_names_transformation/requirements.txt:1]: Please add this to a separate file called [requirements-disabled.txt](https://github.com/GEM-benchmark/NL-Augmenter/blob/fd9b699fc75824db942bdf485f49dc5c31bf60dc/transformations/pinyin/requirements-disabled.txt)

File: transformations/city_names_transformation/test.json [hunk 3]
```diff
@@ -0,0 +1 @@
...ion ."}]}, {"class": "CityNamesTransformation", "inputs": {"sentence": "The team was established in Dallas in 1898 and was a charter member of the NFL in 1920."}, "outputs": [{"sentence": "The team was established in Viera West in 1898 and was a charter member of the NFL in 1920 ."}]}, {"class": "CityNamesTransformation", "inputs": {"sentence": "The Falcons had their first Monday Night Football game in Los Angeles during the 1970 season."}, "outputs": [{"sentence": "The Falcons had their first Monday Night Football game in Old Jefferson during the 1970 season ."}]}, {"class": "CityNamesTransformation", "inputs": {"sentence": "In the event, Manchester was chosen instead of Miami, owing to political circumstances."}, "outputs": [{"sentence": "In the event , Manchester was chosen instead of Hertford , owing to political circumstances ."}]}, {"class": "CityNamesTransformation", "inputs": {"sentence": "In 1841, Sax relocated permanently to Boston and began work on a new set of instruments which were exhibited there in 1844."}, "outputs": [{"sentence": "In 1841 , Sax relocated permanently to Stony Plain and began work on a new set of instruments which were exhibited there in 1844 ."}]}]}
```
Comment (kaustubhdhole) [inline transformations/city_names_transformation/test.json:1]: Maybe this JSON could be formatted a bit..

**Comment by afnanmmir** [comment]
Requested changes have been added!

**Review by mille-s — APPROVED** [review]

**Comment by kaustubhdhole** [comment]
Okay, I am merging this since the build error is fixed.

**Pull request merged**